src/main/java/graphql/execution/incremental/IncrementalExecutionContextKeys.java

```diff
@@ -0,0 +1,26 @@
+package graphql.execution.incremental;
+
+
+import graphql.Internal;
+import org.jspecify.annotations.NullMarked;
+
+/**
+ * GraphQLContext keys for controlling incremental execution behavior.
+ */
+@Internal
+@NullMarked
+public final class IncrementalExecutionContextKeys {
+    private IncrementalExecutionContextKeys() {
+    }
+
+    /**
+     * Enables eager start of @defer processing so defered work runs before the initial result is computed. 
+     * Defaults to false.
+     * <p>
+     * Expects a boolean value.
+     */
+    public static final String ENABLE_EAGER_DEFER_START = "__GJ_enable_eager_defer_start";
+
+}
+
+
```

Comment:
We need to create a new entry in the unusual config - lets put it in

`graphql.GraphQLUnusualConfiguration.IncrementalSupportConfig`

```
        /**
         * This controls whether @defer field execution starts as early as possible.
         */
        @ExperimentalApi
        public IncrementalSupportConfig enableEarlyFieldExecution(boolean enable) {
            contextConfig.put(YOUR_KEY_HERE, enable);
            return this;
        }
```